docs/source/getting-started/content/proc_troubleshooting-bundle-version-mismatch.adoc

```diff
@@ -0,0 +1,34 @@
+[id="troubleshooting-bundle-version-mismatch_{context}"]
+= Troubleshooting bundle version mismatch
+
+As {prod} releases are setup and started, instances are created with bundle information 
```

Comment:
"set up" instead of "setup". This applies generally.

Also, could you please keep each sentence on its own line? We don't particularly care about the line length, but want to keep each sentence isolated so that diffs are cleaner. :)